src/cli/commands/check.js

```diff
@@ -15,6 +15,160 @@ export const noArguments = true;
 
 export function setFlags(commander: Object) {
   commander.option('--integrity');
+  commander.option('--commonjs');
```

Comment:
As @davidaurelio suggested `commonjs` is probably not related to node js modules resolution, let's think of a better name:
- `--resolve-correctly` 
- `--verify-tree` 
- `--check-tree`

other suggestion is to unify `check` and pass a level variable to it, so it will be something like `--check lite` `--check deep` `--check full` or `--check 0`...`--check 2` or as an array (yargs supports this, not sure what commander does) `--check 1 3`